Pod/Core/TypeConversion.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TypeConversion.swift
+//  Pods
+//
+//  Created by Sam Dean on 4/4/17.
+//
+//
+
+import Foundation
+
+
+public protocol FromStringable {
```

Comment:
There's a gotcha with this (and I think it's the reason that `Int` isn't `StringRepresentable`) - if you add `init?(_ string: String)` to Int, you get ambiguous method warnings all over the place because `init?(_ string: String, radix: Int = default)` has a default property on it. Annoyingly,that's enough to make nothing compile, but it _isn't_ enough to let Int confirm to StringRepresentable - SIGH. I'll have a play around and see if I can get something to work :|